service/service.go

```diff
@@ -216,9 +214,14 @@ func (s *Service) deploy() error {
 	s.sendStatus("Image built with success", DDonePositive)
 
 	s.configuration().Image = imageHash
+	// TODO: FOR TEST ONLY
+	if s.Hash == "" {
+		panic("s.Hash is empty")
```

Comment:
needs to be removed?

Reply:
@NicolasMahe why there is this panic here ? Does this code need to be removed ?